h2o-py/h2o/h2o.py

```diff
@@ -320,7 +323,7 @@ def upload_file(path, destination_frame=None, header=0, sep=None, col_names=None
     return H2OFrame()._upload_parse(path, destination_frame, header, sep, col_names, col_types, na_strings)
 
 
-def import_file(path=None, destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
+def import_file(path=None, pattern="", destination_frame=None, parse=True, header=0, sep=None, col_names=None, col_types=None,
```

Comment:
@st-pasha This API already exists in R so I was just adding it to Python. Not sure how it got missed to begin with. But I can try to add file matching from path. However, I will still keep `pattern` as its been a long standing argument for `h2o.importFile` in the R API.